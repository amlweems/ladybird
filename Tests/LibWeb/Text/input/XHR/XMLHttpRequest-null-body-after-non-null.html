<!DOCTYPE html>
<script src="../include.js"></script>
<script>
    asyncTest((done) => {
        const xhr = new XMLHttpRequest()

        function check(method, url, next) {
            xhr.onload = () => {
                println(method + ' ' + xhr.status)
                next()
            }
            xhr.open(method, url, true)
            xhr.send('data')
        }

        check('POST', 'data:text/plain,', () => {
            check('GET', 'https://httpbin.org/status/200', done)
        })
    });
</script>
