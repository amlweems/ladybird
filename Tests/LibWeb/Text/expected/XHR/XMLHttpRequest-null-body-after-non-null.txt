POST 200
GET 200
